convert_fire_to_json.py

```diff
@@ -1073,7 +1074,7 @@ def run(self, filename, assetpath, creatorassets):
 
         self._state._assetpath = assetpath
         self._state._filename = os.path.splitext(os.path.basename(filename))[0]
-        json_name = "json/%s.json" % self._state._filename
```

Comment:
i mean the python script has but, it uses a relative path, should accept parameter to set the path.